.travis.yml

```diff
@@ -53,19 +53,44 @@ script:
 
 stage_osx:
   install: &osx_install
-    # Setup ccache
-    - brew update
-    - brew install ccache
     - export PATH="/usr/local/opt/ccache/libexec:$PATH"
     - export ROBOTOLOGY_ENABLE_DYNAMICS=FALSE
-    # Install dependencies available from Homebrew/core
-    - brew install ace asio eigen boost tinyxml swig qt5 gsl pkg-config jpeg sqlite tinyxml
+    addons:
```

Comment:
I guess the addons need to be not be a child of the `install` YAML element, see where it is placed in YARP travis script: https://github.com/robotology/yarp/blob/master/.travis.yml#L49 .